packages/integration-objects/srv/validations/CheckValidityOfKey.js

```diff
@@ -0,0 +1,24 @@
+/* Copyright (c) 2020 SAP SE or an SAP affiliate company. All rights reserved. */
+const { logger } = require('../logging/Logger');
+const ErrorStatuses = require('../utils/ErrorStatuses');
+const RateTypeExtensionConstants = require('../utils/RateTypeExtensionConstants');
+const { ValidationError } = require('../exceptions/validation-error');
+
+async function checkForValidityOfKey(req, ExchangeRateTypes) {
+  const query = SELECT.from(ExchangeRateTypes).where({ ID: req.params }).limit(1, 0);
+
+  const tx = cds.transaction(req);
+  const affectedRows = await tx.run(query);
+
+  if (affectedRows.length > 0) {
```

Comment:
Minor: you can simplify
```suggestion
  if (affectedRows.length) {
```